python/core/core.sip.in

```diff
@@ -91,21 +91,24 @@ done:
 
 %End
 
+%Feature HAVE_GUI
+%Feature HAVE_QTSERIALPORT
```

Comment:
Do we need all the other features?

Reply:
@m-kuhn  hmm, that's a good question - It was already there... just tell me if you feel some of them should be removed in this PR as well :+1:

Reply:
They were not there, that's why they are green (added) in https://github.com/qgis/QGIS/pull/56953/files

Can you keep this clean?

Reply:
Oh I see, you moved them up

Reply:
@m-kuhn sure they were ... just a few lines lower ... and the need to be introduced before `%If (<your_feature>)` that's why I suggest to shift them all together